tests/conftest.py

```diff
@@ -93,3 +93,29 @@ def adr_service_query(request, pytestconfig: pytest.Config) -> Service:
         tmp_service._container.save_config()
     tmp_service.start(create_db=False, exit_on_close=True, delete_db=False)
     return tmp_service
+
+
+@pytest.fixture
+def adr_template_json(request, pytestconfig: pytest.Config) -> Service:
+    use_local = pytestconfig.getoption("use_local_launcher")
+    local_db = os.path.join("test_data", "template_json")
+    db_dir = os.path.join(request.fspath.dirname, local_db)
+    tmp_docker_dir = os.path.join(
+        os.path.join(request.fspath.dirname, "test_data"), "tmp_docker_json"
+    )
+    if use_local:
+        ansys_installation = pytestconfig.getoption("install_path")
+    else:
+        cleanup_docker(request)
```

Comment:
I merged an improvement to testing, so you may have to pull those and fix the merge conflicts. Also, fixtures are created slightly differently now. Check the new conftest.py.